std_msgs/Header header
string mission_id
sensor_msgs/Image h_frame
sensor_msgs/Image v_frame